Packages.props

```diff
@@ -77,6 +77,7 @@
     <PackageReference Update="System.Text.Encoding.CodePages" Version="8.0.0" />
     <PackageReference Update="System.Text.Encodings.Web" Version="8.0.0" />
     <PackageReference Update="System.Text.Json " Version="8.0.4" />
+    <PackageReference Update="System.Drawing.Common" Version="5.0.3" />
```

Comment:
How is `System.Drawing.Common` currently used? I see the build failed with CG error related to it.

Reply:
I think this package is indirectly referenced by migration nugets. I am force updating it to unblock build pipelines.

Reply:
makes sense